radio/tests/__init__.py

```diff
@@ -18,7 +18,6 @@
 )
 
 
-
 def falcon_test(
```

Comment:
needs return type

Reply:
I have just realised this was _my_ code and I remember why it has no return type so ignore this